esp/templates/program/modules/teacherclassregmodule/coteachers.html

```diff
@@ -74,7 +71,7 @@
 <form action="{{request.path}}" method="post" name="addteacher" onsubmit="cleanTeacherSubmit($j(this))">
 <table align="center" width="500">
     <tr>
-        <th colspan="2">Add More Coteachers</th>
+        <th colspan="2">Add A Coteacher</th>
```

Comment:
Maybe it's just me, but this capitalization looks really weird (especially capitalizing "A").  I recommend to capitalize the first word only.